static/js/util/currency.js

```diff
@@ -1,17 +1,50 @@
 // @flow
 import cc from 'currency-codes';
 import R from 'ramda';
+import iso3166 from 'iso-3166-2';
+
+export const excludedCurrencyCodes = [
```

Comment:
I want to avoid APIs built really quickly unless we have an immediate need for them